builder/builder.go

```diff
@@ -1737,13 +1741,47 @@ func (b *Builder) BuildImage(format string, template string) error {
 		format = b.State.Mix.Format
 	}
 
-	// If the user did not pass in a template, default to release-image-config.json
+	// swupd (client) called by itser will need a temporary directory to act as its stage dir.
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	// If the user did not pass in a template, use the default
 	if template == "" {
-		template = "release-image-config.json"
+		template = imageTemplate
+
+		// If  the default image template is not present in the mix workspace,
+		// copy it from the default ister config directory and update the bundle list based on mix bundles
+		templateFile := filepath.Join(wd, template)
+		if _, err = os.Stat(templateFile); os.IsNotExist(err) {
+			fmt.Printf("Warning: Image template %s not found\n", templateFile)
+
+			configFile := filepath.Join(isterConfigDir, template)
+			fmt.Printf("Copying image template from %s\n", configFile)
+
+			// Read ister template
+			var configValues []byte
+			configValues, err = ioutil.ReadFile(configFile)
+			if err != nil {
+				return err
+			}
+
+			mixBundlesFile := filepath.Join(wd, b.MixBundlesFile)
+			fmt.Printf("Updating image bundle list based on %s\n", mixBundlesFile)
+			if err = UpdateImageBundles(&configValues, mixBundlesFile); err != nil {
+				fmt.Println("Warning: Could not update image bundle list. Using default bundle list instead.")
+			}
+
+			err = ioutil.WriteFile(templateFile, configValues, 0644)
+			if err != nil {
+				return err
+			}
+		} else if err != nil {
+			return err
+		}
 	}
 
-	// swupd (client) called by itser will need a temporary directory to act as its stage dir.
-	wd, _ := os.Getwd()
```

Comment:
Oh yikes, I see the current directory was being used all along.

Regardless, since you are modifying this code let's fix it now.

Reply:
As discussed, not fixing this existing bug in this PR. Created another Issue to track this.